engines/scumm/detection.cpp

```diff
@@ -130,23 +130,7 @@ DetectedGames ScummMetaEngineDetection::detectGames(const Common::FSList &fslist
 }
 
 const char *ScummMetaEngineDetection::getName() const {
-	return "SCUMM ["
```

Comment:
However, this change should not be here. I'll revert it. This shows which subengines were compiled in